xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -104,6 +103,7 @@ CPVRRecording::CPVRRecording(const PVR_RECORDING &recording, unsigned int iClien
   m_bIsDeleted                     = recording.bIsDeleted;
   m_iEpgEventId                    = recording.iEpgEventId;
   m_iChannelUid                    = recording.iChannelUid;
+  SetGenre(recording.iGenreType, recording.iGenreSubType, *recording.strGenreDescription);
```

Comment:
If you change paramter type like I suggested, no need for * in front of recording.strGenreDescription any more.